src/Components/Articles.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+import { fetchArticles } from "../../utils"
+
+function Articles () {
+
+    const [articles, setArticles] = useState([])
+
+    useEffect(() => {
+        fetchArticles().then(({articles}) => {
+            setArticles(articles)
+        })
+    })
+
+    return (
+        <main>
+            <ul>
+            {articles.map((article) => {
+                return <li key={article.article_id}>
```

Comment:
Instead of having all these tags here for an individual article, a nicer way is to create a separate component for that here like an 'ArticleCard' and use that instead to display an article.